pkgs/by-name/wa/waybar-lyric/package.nix

```diff
@@ -0,0 +1,31 @@
+{
+  lib,
+  buildGoModule,
+  fetchFromGitHub,
+}:
+buildGoModule rec {
+  pname = "waybar-lyric";
+  version = "0.10.0";
+
+  src = fetchFromGitHub {
+    owner = "Nadim147c";
+    repo = "waybar-lyric";
+    rev = "v${version}";
+    sha256 = "sha256-glY0W/fB57pYrAhIxYSnkJBmqeN6vy0cwrS71CnKeK4=";
+  };
+
+  # Vendor hash for Go dependencies
+  vendorHash = "sha256-DBtSC+ePl6dvHqB10FyeojnYoT3mmsWAnbs/lZLibl8=";
+
+  # Disable tests
+  doCheck = false;
+
+  # Metadata for the package
+  meta = with lib; {
+    description = "A Waybar module for displaying song lyrics";
+    homepage = "https://github.com/Nadim147c/waybar-lyric";
+    license = licenses.gpl3;
```

Comment:
This is wrong, the license is [AGPL 3.0](https://github.com/Nadim147c/waybar-lyric/blob/main/LICENSE) and not GPL 3.0.